deploy/operator_ironic.yaml

```diff
@@ -0,0 +1,169 @@
+                  name: ironic-bmo-configmap
+                  key: ironic_inspector_endpoint
+        - name: ironic-dnsmasq
+          image: quay.io/metalkube/metalkube-ironic
+          imagePullPolicy: Always
+          securityContext:
+            privileged: true
+          command:
+            - /bin/rundnsmasq
+          volumeMounts:
+            - mountPath: /shared
+              name: ironic-data-volume
+          env:
+            - name: HTTP_PORT
+              valueFrom:
+                configMapKeyRef:
+                  name: ironic-bmo-configmap
+                  key: http_port
+            - name: IP
+              valueFrom:
+                configMapKeyRef:
+                  name: ironic-bmo-configmap
+                  key: http_ip
+            - name: DHCP_RANGE
+              valueFrom:
+                configMapKeyRef:
+                  name: ironic-bmo-configmap
+                  key: dhcp_range
+            - name: INTERFACE
+              valueFrom:
+                configMapKeyRef:
+                  name: ironic-bmo-configmap
+                  key: interface
+        - name: mariadb
+          image: quay.io/metalkube/metalkube-ironic
+          imagePullPolicy: Always
+          securityContext:
+            privileged: true
+          command:
+            - /bin/runmariadb
+          volumeMounts:
+            - mountPath: /shared
+              name: ironic-data-volume
+          env:
+            - name: MARIADB_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: mariadb-password
+                  key: password
+        - name: ironic-httpd
+          image: quay.io/metalkube/metalkube-ironic
+          imagePullPolicy: Always
+          securityContext:
+            privileged: true
+          command:
+            - /bin/runhttpd
+          volumeMounts:
+            - mountPath: /shared
+              name: ironic-data-volume
+          env:
+            - name: HTTP_PORT
+              valueFrom:
+                configMapKeyRef:
+                  name: ironic-bmo-configmap
+                  key: http_port
+            - name: IP
+              valueFrom:
+                configMapKeyRef:
+                  name: ironic-bmo-configmap
+                  key: http_ip
+        - name: ironic
+          image: quay.io/metalkube/metalkube-ironic
+          imagePullPolicy: Always
+          securityContext:
+            privileged: true
+          command:
+            - /bin/runironic
+          volumeMounts:
+            - mountPath: /shared
+              name: ironic-data-volume
+          env:
+            - name: MARIADB_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: mariadb-password
+                  key: password
+            - name: HTTP_PORT
+              valueFrom:
+                configMapKeyRef:
+                  name: ironic-bmo-configmap
+                  key: http_port
+            - name: IP
```

Comment:
We talked today about the need to auto-detect this IP based on a CIDR value, since the pod may move around within the cluster. But let's merge this and iterate on that step, since it will require changes to the image definitions.